File: python_programming_basics/conditional_statements/e_07_shopping.py
budget = float(input())
video_cards_count = int(input())
cpus_count = int(input())
ram_memory_count = int(input())

video_cards_sum = video_cards_count * 250
cpus_sum = cpus_count * video_cards_sum * 0.35
ram_memory_sum = ram_memory_count * video_cards_sum * 0.1

total_sum = video_cards_sum + cpus_sum + ram_memory_sum

if video_cards_count > cpus_count:
    total_sum -= total_sum * 0.15

if budget >= total_sum:
    print(f"You have {budget - total_sum:.2f} leva left!")

else:
    print(f"Not enough money! You need {total_sum - budget:.2f} leva more!")

File: JavaScript Basics/02. Conditional Statements/e-07-Shopping.js
function solve(budget, numGPUs, numCPUs, numRAMs) {
    let gpuPrice = 250;
    let totalGPUPrice = numGPUs * gpuPrice;
    
    let cpuPrice = totalGPUPrice * 0.35;
    let ramPrice = totalGPUPrice * 0.10;

    let totalCPUPrice = numCPUs * cpuPrice;
    let totalRAMPrice = numRAMs * ramPrice;

    let totalPrice = totalGPUPrice + totalCPUPrice + totalRAMPrice;

    if (numGPUs > numCPUs) {
        totalPrice *= 0.85;
    }

    if (budget >= totalPrice) {
        let moneyLeft = budget - totalPrice;
        console.log(`You have ${moneyLeft.toFixed(2)} leva left!`);
    
    } else {
        let moneyNeeded = totalPrice - budget;
        console.log(`Not enough money! You need ${moneyNeeded.toFixed(2)} leva more!`);
    }
}

solve(900, 2, 1, 3);
solve(920.45, 3, 1, 1);
